tests/adapters/compliance_tests/add_interface_to_bond_test.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2015 Internap.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from hamcrest import assert_that, is_, none, empty, has_item, has_properties
+from netman.core.objects.exceptions import UnknownBond
+
+from tests.adapters.configured_test_case import ConfiguredTestCase
+
+
+class AddInterfaceToBondTest(ConfiguredTestCase):
+    _dev_sample = "cisco"
+
+    def test_adding_an_interface_to_a_bond(self):
```

Comment:
The sentence : 
AddInterfaceToBondTest adding an interface to a bond
is not readable, sorry to be an ass about that
Proposition :
test_adds_the_interface_to_the_bond

it's a pedantic issue, yes but if we can manage to have an auto-documentation with these it would be awesome :)